frontend/src/components/ShootDetails/GShootMonitoringCard.vue

```diff
@@ -54,32 +56,50 @@ SPDX-License-Identifier: Apache-2.0
           {{ metricsNotAvailableText }}
         </g-list-item-content>
       </g-list-item>
-    </g-list>
+    </template>
   </v-card>
 </template>
 
 <script setup>
 import { computed } from 'vue'
+import { storeToRefs } from 'pinia'
+
+import { useAuthzStore } from '@/store/authz'
+import { useConfigStore } from '@/store/config'
 
 import GShootStatus from '@/components/GShootStatus'
 import GStatusTags from '@/components/GStatusTags'
 import GClusterMetrics from '@/components/GClusterMetrics'
 
 import { useShootItem } from '@/composables/useShootItem'
+import { useShootHelper } from '@/composables/useShootHelper'
+
+const authzStore = useAuthzStore()
+const {
+  canGetSecrets,
+} = storeToRefs(authzStore)
+
+const {
+  isOidcObservabilityUrlsEnabled,
+} = useConfigStore()
 
+const shootItem = useShootItem()
 const {
   shootNamespace,
   shootName,
   shootConditions,
   isTestingCluster,
-  seedShootIngressDomain,
-} = useShootItem()
+} = shootItem
+
+const {
```

Comment:
I think it cloud makes sense to always provider a single instance of `shootHelper` in the same way as we do it for shootItem?

Reply:
✅  fixed with https://github.com/gardener/dashboard/pull/1899/commits/b2b384256fd979c3729791ce0f6f938547fb8693